src/Nethermind/Nethermind.JsonRpc.Test/Modules/RpcTransaction/BlobTransactionForRpcTests.cs

```diff
@@ -0,0 +1,136 @@
+
+        Build.WithNonce(UInt256.Zero).TestObject,
+        Build.WithNonce(123).TestObject,
+        Build.WithNonce(UInt256.MaxValue).TestObject,
+
+        Build.WithTo(null).TestObject,
+        Build.WithTo(TestItem.AddressA).TestObject,
+        Build.WithTo(TestItem.AddressB).TestObject,
+        Build.WithTo(TestItem.AddressC).TestObject,
+        Build.WithTo(TestItem.AddressD).TestObject,
+        Build.WithTo(TestItem.AddressE).TestObject,
+        Build.WithTo(TestItem.AddressF).TestObject,
+
+        Build.WithGasLimit(0).TestObject,
+        Build.WithGasLimit(123).TestObject,
+        Build.WithGasLimit(long.MaxValue).TestObject,
+
+        Build.WithValue(UInt256.Zero).TestObject,
+        Build.WithValue(123).TestObject,
+        Build.WithValue(UInt256.MaxValue).TestObject,
+
+        Build.WithData(TestItem.RandomDataA).TestObject,
+        Build.WithData(TestItem.RandomDataB).TestObject,
+        Build.WithData(TestItem.RandomDataC).TestObject,
+        Build.WithData(TestItem.RandomDataD).TestObject,
+
+        Build.WithMaxPriorityFeePerGas(UInt256.Zero).TestObject,
+        Build.WithMaxPriorityFeePerGas(123).TestObject,
+        Build.WithMaxPriorityFeePerGas(UInt256.MaxValue).TestObject,
+
+        Build.WithMaxFeePerGas(UInt256.Zero).TestObject,
+        Build.WithMaxFeePerGas(123).TestObject,
+        Build.WithMaxFeePerGas(UInt256.MaxValue).TestObject,
+
+        Build.WithMaxFeePerBlobGas(UInt256.Zero).TestObject,
+        Build.WithMaxFeePerBlobGas(123).TestObject,
+        Build.WithMaxFeePerBlobGas(UInt256.MaxValue).TestObject,
+
+        Build.WithAccessList(new AccessList.Builder()
+            .AddAddress(TestItem.AddressA)
+            .AddStorage(1)
+            .AddStorage(2)
+            .AddStorage(3)
+            .Build()).TestObject,
+
+        Build.WithAccessList(new AccessList.Builder()
+            .AddAddress(TestItem.AddressA)
+            .AddStorage(1)
+            .AddStorage(2)
+            .AddAddress(TestItem.AddressB)
+            .AddStorage(3)
+            .Build()).TestObject,
+
+        Build.WithBlobVersionedHashes(0).TestObject,
+        Build.WithBlobVersionedHashes(1).TestObject,
+        Build.WithBlobVersionedHashes(50).TestObject,
+
+        Build.WithChainId(null).TestObject,
+        Build.WithChainId(BlockchainIds.Mainnet).TestObject,
+        Build.WithChainId(BlockchainIds.Sepolia).TestObject,
+        Build.WithChainId(0).TestObject,
+        Build.WithChainId(ulong.MaxValue).TestObject,
+
+        Build.WithSignature(TestItem.RandomSignatureA).TestObject,
+        Build.WithSignature(TestItem.RandomSignatureB).TestObject,
+    ];
+
+    public static void ValidateSchema(JsonElement json)
+    {
+        // NOTE: This does not look correct since we are not checking for the exact value (`0x3`).
+        // Essentially we're accepting all hex values `0x00 ... 0xFF`, including `0x`
+        json.GetProperty("type").GetString().Should().MatchRegex("^0x([0-9a-fA-F]?){1,2}$");
```

Comment:
According to the spec (https://github.com/ethereum/execution-apis/blob/71260dde2d7fc5a12b9d171692a19a5e599820cf/src/schemas/transaction.yaml#L20) any hex byte should be acceptable.

I think this is a mistake and instead it should be defined to be a specific value like in, for example, EIP1559 (https://github.com/ethereum/execution-apis/blob/71260dde2d7fc5a12b9d171692a19a5e599820cf/src/schemas/transaction.yaml#L93-L96)

Reply:
Replaced with `^0x3$` since it's most likely an error in the spec.